engines/sci/resource.cpp

```diff
@@ -980,8 +982,12 @@ void ResourceManager::init() {
 #ifdef ENABLE_SCI32
 	_currentDiscNo = 1;
 #endif
-	_patcher = new ResourcePatcher(g_sci->getGameId(), g_sci->getLanguage());
-	addSource(_patcher);
+	if(g_sci) {
```

Comment:
Please fix spacing here, to conform to our coding guidelines

Reply:
We're talking about the `if(g_sci)` that should be `if (g_sci)`, right?
Fixed.